libsolidity/formal/CVC4Interface.cpp

```diff
@@ -137,6 +137,12 @@ CVC4::Expr CVC4Interface::toCVC4Expr(Expression const& _expr)
 				return m_context.mkConst(true);
 			else if (n == "false")
 				return m_context.mkConst(false);
+			else if (_expr.sort->kind == Kind::Sort)
+			{
+				auto sortSort = dynamic_pointer_cast<SortSort>(_expr.sort);
+				solAssert(sortSort, "");
```

Comment:
Is this important for debugging? Otherwise it could be
```suggestion
				if (auto sortSort = dynamic_pointer_cast<SortSort>(_expr.sort))
				    return m_context.mkVar(n, cvc4Sort(*sortSort->inner));
```

Reply:
Well, it should assert if the sort is wrong.

Reply:
But you're only asserting that it's not `null`, right? But I'm fine with how it is.

Reply:
Yes, but in case the `if` fails, it should assert false because there should be no other possibility